paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheet.kt

```diff
@@ -1577,13 +1577,13 @@ class PaymentSheet internal constructor(
         @ExtendedAppearancePreview
         constructor(
             context: Context,
-            cornerRadiusDp: Int,
-            borderStrokeWidthDp: Int,
-            bottomSheetCornerRadiusDp: Int,
+            @DimenRes cornerRadiusDimen: Int,
```

Comment:
Sorry... Can we make this cornerRadiusRes to match https://github.com/stripe/stripe-android/pull/10885/files#diff-ca1efb8b1b52c4cd41cee8b0ac95ed4fe835b3736bc02cf73fd53dfdc7ba07f0R1716